pglookout/pglookout.py

```diff
@@ -452,7 +452,7 @@ def check_cluster_state(self):
             self.consider_failover(own_state, master_node, standby_nodes)
 
     def consider_failover(self, own_state, master_node, standby_nodes):
-        if not master_node:
+        if not master_node or not master_node.get("connection"):
```

Comment:
The change looks really reasonable.
Just one thing - the new condition does not match the log entry below. There could be a master node in the state, but no connection.
Probably it would make sense to change the log message to include this info so it will be easier to debug, or have another log entry for this case.

Reply:
I've updated the logging to reflect what the current state is and also indicate if the decision was taken because of a re-configuration.